src/main/java/duke/tasks/Task.java

```diff
@@ -0,0 +1,44 @@
+package duke.tasks;
+
+/**
+ * Stores inputs of tasks
+ */
+public class Task {
+    private final String name;
+    private boolean checkMark;
+
+    /**
+     * Initialises input to the class
+     *
+     * @param name name of the tasks
+     */
+    public Task(String name) {
+        this.name = name;
+        this.checkMark = false;
+    }
+
+    /**
+     * Sets checkMark as true
+     */
+    public void toBeMarked() {
+        this.checkMark = true;
+    }
+
+    /**
+     * Sets checkMark as false
+     */
+    public void toBeUnmarked() {
+        this.checkMark = false;
+    }
```

Comment:
I think that renaming these functions as verbs might help make their purpose clearer